optional_plugins/runner_remote/avocado_runner_remote/__init__.py

```diff
@@ -0,0 +1,670 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: Red Hat Inc. 2017
+# Author: Cleber Rosa <crosa@redhat.com>
```

Comment:
Actually thinking about it a bit more I think the `Remoter` should be separated from this plugin. If you remember I said elsewhere that it's actually being used inside tests. I'd really prefer it to become a separate plugin, ideally extending the `avocado.utils`. Alternatively just a different namespace. Anyway as this will require changes to existing tests (downstream) I think we should consider the solution first before pushing intermittent version and changing it twice in a row.